### DISCUSSION TITLE
--notify flag added in latest Shopify CLI v3.46!

### DISCUSSION THREAD

**Discussion by knjshimi** [barrel/shopify-vite#80] [General]
Hey @montalvomiguelo, you probably know this, but anyway - your work on the `--notify` flag has been shipped with v3.46! 🥳  Great stuff mate!

**Comment by montalvomiguelo** [comment]
Yay! @kenji-twp, we can do something cool with that. e.g. _A vite plugin that watches the notification to hot reload the theme editor after editing theme files like json schemas, locales, etc._

**Comment by montalvomiguelo** [comment]
Just updated this [theme-extension](https://github.com/montalvomiguelo/theme-app-extension-vite) to use the `notify` flag to handle live reloading with [vite-plugin-full-reload](https://github.com/ElMassimo/vite-plugin-full-reload/tree/main) whenever there's a tweak in the extension files 🎉

https://github.com/barrel/shopify-vite/assets/5134470/0021dd71-9f9c-4c08-acab-bf5d2adbefe3

**Reply by knjshimi** [reply]
That's so cool @montalvomiguelo!! Thanks heaps for that :) I've seen you also fixing cors issues and other contributions to shopify cli, great work mate!

So that opens the doors for theme editor HMR as well (is that already covered)?

**Comment by montalvomiguelo** [comment]
Thanks @kenji-twp (: 

Yup, HMR for assets like scripts/styles is covered; this will help us livereload the theme editor when theme/extension files (json, liquid, etc) change. No more manual refreshes (:

**Comment by ysemennikov** [comment]
Hi @montalvomiguelo @wturnerharris,
Hope you are doing well. We have discussed the latest fix about 2 days ago. Haven't you unpublished the latest package version from NPM? 2.1.3 was available about 2-3 hrs ago, but now the latest version is 2.1.1 (as it was before).
